### PR TITLE
Push to rubygems

### LINKED ISSUE #14: Add automatic push to rubygems
In a GH action
Can be inspired of https://github.com/meilisearch/meilisearch-ruby/blob/main/.github/workflows/gempush.yml

### PR DESCRIPTION
**Pull request by CaroFG**
Closes #14

### CODE REVIEW

**Review by curquiza — CHANGES_REQUESTED** [review]
This is my fault, it's something I will change in the ruby SDK, but can you create a `.github/scripts/` folder where you will put the script? Instead of storing the script in `/scripts/` folder at the root of the repo.

File: .github/workflows/gempush.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    tags:
+      - v*
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
```
Comment (curquiza) [inline .github/workflows/gempush.yml:14]:
```suggestion
    - uses: actions/checkout@v2
```

Comment (curquiza) [reply .github/workflows/gempush.yml:14]: This is better to "control" what we use, I need to change this is all the other repos.

File: .github/workflows/gempush.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
```
Comment (curquiza) [inline .github/workflows/gempush.yml:11]:
```suggestion
    runs-on: ubuntu-20.04
```

**Comment by CaroFG** [comment]
bors merge

**Pull request merged**